stardist_napari/_dock_widget.py

```diff
@@ -132,7 +133,7 @@ class Output(Enum):
         perc_high    = 99.8,
         prob_thresh  = 0.5,
         nms_thresh   = 0.4,
-        output_type  = Output.Both.value,
+        output_type  = Output.Labels.value,
```

Comment:
I'm not happy with this, because polygon output is nice and people may not discover it if it's not on by default.
On the other hand, polygon support in napari/vispy is currently very slow if there are many of them...

Reply:
> I'm not happy with this, because polygon output is nice and people may not discover it if it's not on by default.

Fully agree!

> On the other hand, polygon support in napari/vispy is currently very slow if there are many of them...

Yes, that was the motivation to disable it by default. For many polygons, such as for long timelapses, the shape layer construction is vastly dominating the plugin runtime making everything very slow.

Reply:
Any advice on how to handle this, @tlambert03 @jni?

Reply:
Seems to be a known issue https://github.com/napari/napari/issues/2015
I think one needs to change the underlying opengl/vispy structures to allow for merged polygonal meshes...

Reply:
Warning okay?

<img width="508" alt="image" src="https://user-images.githubusercontent.com/231996/142009677-b0143260-1652-4278-b170-04640d4a0a33.png">

Reply:
Yip! nice color! :)